website/views/options/social/group.jade

```diff
@@ -18,7 +18,7 @@ a.pull-right.gem-wallet(ng-if='group.type!="party"', popover-trigger='mouseenter
           h3.panel-title
             | {{group.name}}
             span.group-leave-join(ng-if='group')
-              a.btn.btn-sm.btn-danger.pull-right(ng-if=":: isMemberOfGroup(User.user._id, group)", ng-click='clickLeave(group, $event)')
+              a.btn.btn-sm.btn-danger.pull-right#btn-leave(ng-if=":: isMemberOfGroup(User.user._id, group)", ng-click='clickLeave(group, $event)')
```

Comment:
I think it would be better to remove the extra styling and the id and just add the `btn-sm` class to the edit and save buttons. That'll fix the alignment and the size discrepancies of the buttons.